docs/Dialects/Seq/RationaleSeq.md

```diff
@@ -158,7 +162,7 @@ end
 ```
 
 Additionally, `sv` operations are also included to provide the register with
-a randomized preset value.
+a randomized preset value, unless an explicit preset is specified.
 Since items assigned in an `always_ff` block cannot be initialised in an
 `initial` block, this register lowers to `always`.
```

Comment:
A slightly different formulation has the preset subsume the randomized preset value. I.e., there is a `preset <value | "random">` and the presence of the "random" flavor is what produces the existing lowering. If no preset is specified, then this is equivalent to disabling randomization.

The main benefit of this is that it seems odd to lump eliding presets under disabling randomization and this provides a way out of it.